Fixed log in + first demo of addIssue connected to backend

NrdKejsi opened the pull request:
Login works, cookies still trying to figure out
Ignore the single error, I'm still in the progress of moving from mock data to real data from server and making sure everything matches, doesn't affect rest of code

Nikolay-Ts reviewed (changes requested):
- remove the package-lock.json
- remove src/app.tsx
- remove the Componnent folder
- will ask rumen to review the functionality with the backend
- why is utils/types being modified

NrdKejsi commented:
1) types.ts I had to only rename everything 'task' into 'issue' cause i don't want both names for the same thing, i changed the rest of the codebase that had 'task' as well. the structures themselves are the same.
2)removed src/app.tsx and Component folder
3)For package-lock.json, do you want me to delete it from my branch?? I thought it being on git.ignore was enough. lmk

Nikolay-Ts commented:
3) I think remove it just in case. IDK why it is still being pushed...

Nikolay-Ts reviewed (approved):
![](https://duckduckgo.com/?q=approved+knuckles+meme&t=osx&ia=images&iax=images&iai=https%3A%2F%2Fi.imgflip.com%2F7bqii3.jpg)
